DEPS

```diff
@@ -236,7 +239,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + 'abada33190daa7914938ec56d59032012fcf12f7',
+  'src': 'https://github.com/flutter/buildroot.git' + '@' + '6d0e54005fac0e5af1a0bbe2426ef49fcb94be28',
```

Comment:
This should be updated to flutter/buildroot tip-of-tree, which is currently https://github.com/flutter/buildroot/commit/c77cac9502fb108be6a1ed4e0e4e1c65649b3a35